### PR TITLE
Updated - concatenation methods

### PR DESCRIPTION
**Pull request by Srivalli-Kolla**

### CODE REVIEW

File: Reading_files/vascular.ipynb [hunk 1]
```diff
@@ -1,5 +1,16 @@
 {
```
Comment (cartal) [inline Reading_files/vascular.ipynb]: <p>1) Please use a more descriptive and serious header. "Huge" is not a thing, and sample wise is written "sample-wise".</p><p>2) Each section should be a bullet point.</p><p>3) You also need to add when the notebook was created and when was the last time it was modified. </p><p> </p><p>For example:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">### Notebook for the concatenation of multiple anndata files into a single one for cardiac pericytes.

- **Developed by**: Srivalli Kolla
- **Würzburgh Institute for Systems Immunology - JMU Würzburg**
- **Created on**: 240408
- **Last modified**: 240408
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/#comment-1555376213'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='0'/>

Comment (cartal) [inline Reading_files/vascular.ipynb]: <p>1) You are importing glob, but I don't see where you are using it. Copuld you specify?</p><p>2) What's the difference between a sparse and dense matrix?</p><p>3) The importing packages section and the setting variable sections has to be separated. </p><p>4) Why are you using such a small image resolution? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/#comment-1555376217'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (cartal) [inline Reading_files/vascular.ipynb]: <p>The table is nice, but this doesn't tell you much as you can't see the whole thing. Could you make a graphic instead? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/#comment-1555376219'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='6'/>

Comment (cartal) [inline Reading_files/vascular.ipynb]: <p>Very nice! </p><p> </p><p>However:</p><p> </p><p>1) you are defeating the purpose of reading from "../Data/" by giving the full path to your hard drive. </p><p>2) I asked you to add a time stamp to the files, so we can keep track of the version of files. Please do.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/#comment-1555376220'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='17'/>

Comment (cartal) [inline Reading_files/vascular.ipynb]: <p>Why did you use <code>join='outer'</code>?  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Heart_Vascular/pull/8/discussion/#comment-1555376222'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='27'/>

**Comment by cartal** [comment]
I added comments to the review of the code, please take a look and prepare for them. 

Don't solve them until we have our 1:1 meeting today :)

**Comment by Srivalli-Kolla** [comment]
Hi @cartal , I have made the changes according to the requirements and have committed in `Updated - All changes`

Please have a look

**Pull request merged**